MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet private(set) weak var errorView: ErrorView!
```

Comment:
This property doesn't need to be `weak`.

```suggestion
	@IBOutlet private(set) var errorView: ErrorView!
```